simulator/risc_v/t/unit_test.cpp

```diff
@@ -313,3 +313,60 @@ TEST_CASE("RISCV RV64 bfp")
     instr.execute();
     CHECK( instr.get_v_dst() == 0x0000'555C'5CCC'0000);
 }
+
+TEST_CASE ("RISCV RV32 clz")
+{
+    CHECK ( RISCVInstr<uint32>(0x60079793).get_disasm() == "clz $a5, $a5");
+    std::vector<TestData<uint32>> cases = {
+        TestData<uint32>( 0 , 0 , 32),
+        TestData<uint32>( -1, 0, 0),
```

Comment:
I'm afraid `-1` cannot be casted to unsigned correctly. Could you please use `all_ones` instead?

Reply:
ok. 5 min

Reply:
There is no need to close and open a new pull request, you can push commits to your branch and they'll be automatically fetched here.

Reply:
done